frontend/Controls/SearchBox.qml

```diff
@@ -1,55 +1,90 @@
-import QtQuick 2.0
+import QtQuick 2.7
 import QtQuick.Layouts 1.3
 import QtQuick.Controls 1.4
 import QtQuick.Controls.Styles 1.4
 import "../Controls" as Controls
 
-Rectangle {
-    id: root
+Item {
+    id: searchBoxWithResults
+
+    property alias text: searchField.text
+    property alias placeholder: searchField.placeholderText
+
     anchors.verticalCenter: parent.Center
     Layout.fillWidth: true
     Layout.maximumWidth: 367
     Layout.minimumWidth: 367
     Layout.preferredWidth: 367
-    height: 44
-    color: "#3A3E46"
-    radius: 5
 
-    property alias text: searchField.text
-    property alias placeholder: searchField.placeholderText
-    signal buttonClicked
+    anchors.top: parent.top
+    anchors.left: parent.left
+    anchors.topMargin: 13
+
+    z: 50
 
     Rectangle {
-        id: searchFieldContainer
-        Layout.fillWidth: true
+        id: searchBoxId
+
+        width: parent.width
         height: 44
-        width: 367
-        color: "transparent"
-        anchors.top: parent.top
-        anchors.left: parent.left
         radius: 5
 
-        TextField {
-            id: searchField
-            height: parent.height
-            width: 220
-            y: 0
+        color: "#3A3E46"
+
+        signal buttonClicked
 
-            font.pointSize: 14
-            font.family: "Roboto"
-            placeholderText: placeholder
+        Rectangle {
+            id: searchFieldContainer
+            Layout.fillWidth: true
+            height: 44
+            width: 367
+            color: "transparent"
```

Comment:
I've since learned this can be achieved more readily by using the QML `Item` type rather than rectangle which have no background fill

Reply:
Done